minio/umbrel-app.yml

```diff
@@ -0,0 +1,34 @@
+manifestVersion: 1.1
+id: minio
+category: developer
+name: MinIO
+version: "RELEASE.2024-02-17T01-15-57Z"
+tagline: The Object Store for AI Data Infrastructure
+description: >-
+    MinIO is a high-performance, S3 compatible object store. It is built for
+    large scale AI/ML, data lake and database workloads. It is software-defined
+    and runs on any cloud or on-premises infrastructure. MinIO is dual-licensed
+    under open source GNU AGPL v3 and a commercial enterprise license.
+
+    Understand the Features that Make MinIO the Industry's Most Widely Deployed Object Store:
+    - Bucket & Object Immutability
+    - Encryption
+    - Active Active Replication for Object Storage
+    - Automated Data Management Interfaces
+    - Object Storage Built for S3
+    - Data Life Cycle Management & Tiering
+    - Scalable Object Storage
+    - Identity & Access Management
```

Comment:
Rendering the yaml nicely is a bit tricky. To get a blank line for whitespace between paragraphs you need to add **2** blank lines in the yaml.

And to make sure the bullet points show up on separate lines you need to add **1** blank line between each.

Otherwise it shows up like this in the UI:

<img width="713" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/28efbc9e-fb69-4647-9cfe-bb0f055f05b0">